### PR TITLE
Basic implementation of uploading files to a s3 bucket

### PR DESCRIPTION
**Pull request by I2olanD**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -16,13 +16,25 @@ Install dependencies
...
+ To make sure that the installation went fine. Run following command.
+
+ ```
+ yarn test
+ ```
+
 ### Usage
 
 After installing this package, introduce this component as followed.
+Make sure that you provide an presignedUrlEndpoint or a presignedUrlEndpointCallback function.
```
Comment (lightblader) [inline readme.md]: provide **a** presignedUrlEndpoint

**Comment by jayhof** [comment]
I fixed the eslint configuration, .vue files did not get linted. After fixing the configuration issues I of course fixed linter offenses. From now on you won't be able to push without .js and .vue files satisfying our linter.

File: src/components/VueBucketLoader.vue [hunk 2]
```diff
@@ -123,7 +121,7 @@ export default {
     },
 
     async uploadFile(presignedUrl, formData, config) {
-      return await axios.post(presignedUrl.data.postEndpoint, formData, config);
```
Comment (jayhof) [inline src/components/VueBucketLoader.vue]: Be careful you returned the resolved promise, then used await again when calling uploadFile.

**Comment by jayhof** [comment]
I'm really unhappy about the docs/webpack.config.babel.js, we have lots of duplicated configuration with our root webpack.config.babel.js

It should be possible to define a global webpack configuration then extending/overwriting the parts we need to change for the docs. (https://www.npmjs.com/package/webpack-merge)

**Pull request merged**